src/Sarif.Converters/CppCheckError.cs

```diff
@@ -31,11 +31,6 @@ internal class CppCheckError
         /// <param name="locations">The locations to which the error generated by CppCheck refer.</param>
         public CppCheckError(string id, string message, string verboseMessage, string severity, ImmutableArray<CppCheckLocation> locations)
         {
-            if (locations.IsDefaultOrEmpty)
-            {
-                throw new ArgumentException(ConverterResources.CppCheckMissingLocation);
```

Comment:
>CppCheckMissingLocation [](start = 63, length = 23)

If this resource is now unused, remove it.

Reply:
Why is this "By Design"? Is the resource used elsewhere?

---
In reply to: [506538800](https://github.com/microsoft/sarif-sdk/pull/2119#discussion_r506538800) [](ancestors = 506538800)

Reply:
updating!

---
In reply to: [506609059](https://github.com/microsoft/sarif-sdk/pull/2119#discussion_r506609059) [](ancestors = 506609059,506538800)

Reply:
I see you removed an unused resource CppCheckCppCheckElementMissing, which is fine. But the resource here is CppCheckMissingLocation, which you can (also) remove -- if indeed it is not used elsewhere.

---
In reply to: [506618367](https://github.com/microsoft/sarif-sdk/pull/2119#discussion_r506618367) [](ancestors = 506618367,506609059,506538800)

Reply:
Looking again, I saw that this is used by other places. so I'm reverting it.

---
In reply to: [506620738](https://github.com/microsoft/sarif-sdk/pull/2119#discussion_r506620738) [](ancestors = 506620738,506618367,506609059,506538800)